tests/core/dom/element/isdetached.js

```diff
@@ -0,0 +1,43 @@
+/* bender-tags: editor,dom */
+( function() {
+	var ancestor = new CKEDITOR.dom.element( 'div' ),
+		parent = new CKEDITOR.dom.element( 'div' ),
+		element = new CKEDITOR.dom.element( 'div' ),
+		body = CKEDITOR.document.getBody();
+
+	bender.test( {
```

Comment:
I don't like the fact that all tests reuses the same set of elements. To be 100% sure that nothing is interfering with test's results, we should create separate DOM structures for every test.